beta-20161103.md

```diff
@@ -0,0 +1,72 @@
+---
+title: What's New in beta-20161103
+toc: false
+summary: Additions and changes in CockroachDB version beta-20161103.
+---
+
+## Oct 27, 2016
+
+### Binaries
+
+<div id="os-tabs" class="clearfix">
+    <a href="https://binaries.cockroachdb.com/cockroach-beta-20161103.darwin-10.9-amd64.tgz"><button id="mac" data-eventcategory="mac-binary-release-notes">Mac</button></a>
+    <a href="https://binaries.cockroachdb.com/cockroach-beta-20161103.linux-amd64.tgz"><button id="linux" data-eventcategory="linux-binary-release-notes">Linux</button></a>
+</div>
+
+### Backwards-Incompatible Changes
+
+- Users (other than `root`) must now be created (with the `CREATE USER` statement or `cockroach user set` command) before being used. [#9794](https://github.com/cockroachdb/cockroach/pull/9794)
```

Comment:
I guess I'm not clear on what "before being used" means. For example, I just successfully granted a non-existing user a privilege on the db and didn't get an error...

```
root@:26257> grant select on database bank to pablo;
GRANT
```